helm-charts/aerospike-cluster/README.md

```diff
@@ -9,34 +9,46 @@ A Helm chart for `AerospikeCluster` custom resource to be used with the Aerospik
 
 ## Usage
 
-### Add Aerospike Helm Repository
+<!-- ### Add Aerospike Helm Repository
 
 ```sh
 helm repo add aerospike https://aerospike.github.io/aerospike-kubernetes-operator
+``` -->
+
```

Comment:
You can add about cert-manager in operator's helm. I think, it's missing there

Reply:
Will add.